.github/workflows/README.md

```diff
@@ -0,0 +1,134 @@
+# Snyk
+
+We have two Snyk actions. [sbt-node-snyk](sbt-node-snyk.yml) and [sbt-node-snyk-pr](sbt-node-snyk-pr.yml).
+
+It is recommended you use both on your project for greater coverage.
+
+## Requirements
+
+1. `SNYK_TOKEN` must be available in your repo's 'Actions secrets' under 'Organization secrets' (Go to `/settings/secrets/actions`). If you cannot see this, ask an engineering manager to add it to your repo.
+1. `with: ORG:` be sure to edit this in the `yml` examples below. This will be the org ID within Snyk.
+![image](https://user-images.githubusercontent.com/48949546/112194614-f6985880-8c00-11eb-946f-a88fdae57662.jpg)
```

Comment:
```suggestion
1. `with: ORG:` be sure to edit this in the `yml` examples below. This will be the org ID within Snyk. [Organisations](https://docs.snyk.io/introducing-snyk/snyks-core-concepts/groups-organizations-and-users#snyk-organizations) are contained within the main `guardian` group and often correspond to teams. You can [create a new organisation](https://docs.snyk.io/features/user-and-group-management/managing-groups-and-organizations/manage-snyk-organizations) if yours doesn't exist already.
![image](https://user-images.githubusercontent.com/48949546/112194614-f6985880-8c00-11eb-946f-a88fdae57662.jpg)
```